src/sync.cpp

```diff
@@ -513,9 +513,10 @@ bool assignFilesystemIds(Sync& sync, MegaApp& app, FileSystemAccess& fsaccess, h
 
 // new Syncs are automatically inserted into the session's syncs list
 // and a full read of the subtree is initiated
-Sync::Sync(MegaClient* cclient, string* crootpath, const char* cdebris,
+Sync::Sync(MegaClient* cclient, SyncConfig config, string* crootpath, const char* cdebris,
            string* clocaldebris, Node* remotenode, fsfp_t cfsfp, bool cinshare, int ctag, void *cappdata)
-    : localroot(new LocalNode)
+: localroot{new LocalNode}
```

Comment:
the round brackets that were already here definitely call a constructor - curlies add a lot more possibilities.  Why change something that's perfectly fine, and possibly introduce problems?

Reply:
Will revert.